.github/workflows/php.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                presta-versions: ['1.7.4.0', '8.0.4']
+                presta-versions: ['1.7.4.0', '8.0.4', '8'] # These are prestashop/prestashop Docker tag versions
```

Comment:
Interesting suggestion.
At this step we prefer launch phpstan for only two defined PrestaShop release.

Reply:
If we add « 8 », « 8.0.4 » is probably not pertinent if PHP version of the docker images are the same.
Why do you choose 8 and not latest ?

Reply:
I was afraid that `latest` might be the 9 development. If there is a convention (undocumented) that latest means the latest release version then I agree that is best.